src/sfctl/commands.py

```diff
@@ -217,6 +217,22 @@ def load_command_table(self, args): #pylint: disable=too-many-statements
                 group.command('get', 'get_property_info')
                 group.command('delete', 'delete_property')
 
+            with super_group.group('mesh app') as group:
+                group.command('show', 'get_application_resource')
+                group.command('delete', 'delete_application_resource')
+
+            with super_group.group('mesh volume') as group:
+                group.command('show', 'get_volume_resource')
+                group.command('delete', 'delete_volume_resource')
+
+            with super_group.group('mesh service') as group:
+                group.command('list', 'get_services')
```

Comment:
Can you change this and the 3 other below to the format get_mesh_services/replicas? I know it's in a different file, but it's a bit clearer IMO what it is